Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+black = "==18.9b0"
+flake8 = "*"
+mypy = "*"
+pylint = "*"
+pytest = "*"
+pytest-cov = "*"
+pytest-django = "*"
+
+[packages]
+Django = "*"
+psycopg2 = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
I would change it to `3.7` :) @rwrzesien are you using `Ubuntu`?

Reply:
Yes, but if you want we can update, I will just have to add another Python version, no problem with that (or at least none that I expect). Or leave it like this for now if you want.